include/Vector2.hpp

```diff
@@ -13,16 +13,21 @@
 namespace GPM
 {
 
-struct Vector2
+union Vector2
 {
     // Data members
-    f32 x;
-    f32 y;
+    struct { f32 x; f32 y; };
+    f32 e[2];
+
+    Vector2() = default;
+    constexpr Vector2(const f32 k)                              noexcept;
+    constexpr Vector2(const f32 x, const f32 y = .0f)           noexcept;
+    constexpr Vector2(const f32 coef[2])                        noexcept;
     
     // Static methods (pseudo-constructors)
-    static constexpr Vector2 zero   () noexcept;
-    static constexpr Vector2 right  () noexcept;
-    static constexpr Vector2 up     () noexcept;
+    static constexpr Vector2 zero   ()                          noexcept;
```

Comment:
Vector2 one can be usefull

Reply:
Fuction to rotate 2D vector can be cool (90, 180, 270 and X angle)

Reply:
`Vector2::one()` has been added.

`Vector2::rotated90()` already exists, and 180° and 270° degrees rotations of a vector `v` can be obtained with `-v` and `-v.rotated90()`, respectively.

`Vector2:rotated(value)` is already implemented for other angles